packages/synapse-interface/components/Activity/AirdropRewards.tsx

```diff
@@ -227,8 +227,9 @@ const RewardsDialog = ({
             className="px-1 overflow-y-scroll"
             style={{ maxHeight: maxHeight }}
           >
-            {transactions.map((transaction) => (
+            {transactions.map((transaction, key) => (
```

Comment:
I know we do this in a few spots (using map index as key), but just found out about performance [implications](https://www.developerway.com/posts/react-key-attribute#part3)

Think we can just use `transaction.transactionHash` as the key since it's unique

Reply:
Super good read, thanks! Agreed that using `transaction.transactionHash` as key and will be careful about assigning `key` properties in array callback fns

Reply:
Updated in a6a73ea7bb6a29ceefb64cf239e4d5ab7775976a